apps/raptor_service/lib/raptor_service.ex

```diff
@@ -102,7 +102,7 @@ defmodule RaptorService do
   end
 
   def check_auth0_role(raptor_url, api_key, role) do
-    case HTTPoison.get(url_for_checking_role(raptor_url, api_key, role) |> IO.inspect(label: "RYAN - URL")) do
```

Comment:
real quick IO inspect

Reply:
Uhhh, that's not mine? I've never seen that IO inspect in my life!